Applications/AdminApi/test/AdminApi.Tests.Integration/StepDefinitions/TokensStepDefinitions.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Backbone.AdminApi.Sdk.Endpoints.Identities.Types.Responses;
+using Backbone.AdminApi.Sdk.Endpoints.Tokens.Response;
+using Backbone.AdminApi.Sdk.Services;
+using Backbone.AdminApi.Tests.Integration.Configuration;
+using Backbone.AdminApi.Tests.Integration.Extensions;
+using Backbone.BuildingBlocks.SDK.Endpoints.Common.Types;
+using Microsoft.Extensions.Options;
+using PaginationFilter = Backbone.BuildingBlocks.SDK.Endpoints.Common.Types.PaginationFilter;
+
+
+namespace Backbone.AdminApi.Tests.Integration.StepDefinitions;
+
+[Binding]
+[Scope(Feature = "GET /Tokens?createdBy={identity-address}")]
+internal class TokensStepDefinitions(HttpClientFactory factory, IOptions<HttpClientOptions> options) : BaseStepDefinitions(factory, options)
+{
+    private ApiResponse<CreateIdentityResponse>? _createIdentityResponse;
```

Comment:
Why do we need a field for that? As far as I can see, it's only used in one method.